tests/AppBundle/EventListener/RedirectToPreferredLocaleListenerTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Tests\AppBundle\EventListener;
+
+use AppBundle\EventListener\RedirectToPreferredLocaleListener;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+/**
+ * Unit test for the event listener.
+ *
+ * See http://symfony.com/doc/current/book/testing.html#unit-tests
+ *
+ * Execute the application tests using this command (requires PHPUnit to be installed):
+ *
+ *     $ cd your-symfony-project/
+ *     $ ./vendor/bin/phpunit
+ */
+class RedirectToPreferredLocaleListenerTest extends \PHPUnit_Framework_TestCase
+{
+    private $requestProphecy;
+    private $urlGeneratorProphecy;
+    private $eventProphecy;
+    private $listener;
+
+    public function setUp()
+    {
+        $this->requestProphecy = $this->prophesize(Request::class);
```

Comment:
I don't use Prophecy so I'm asking: is it common to add the `Prophecy` suffix to the * prophesized* variables?